### PR TITLE
Add search button to agency component preview

### PR DESCRIPTION
**Pull request by hbillings**
I have thoughts I'd love feedback on over in #282.

### CODE REVIEW

**Comment by adborden** [comment]
If we did this, I think it should be an actual button element.

**Comment by adborden** [comment]
@hbillings I just pushed a change, let me know what you think.

File: js/components/agency_component_selector.jsx [hunk 1]
```diff
@@ -85,20 +85,32 @@ class AgencyComponentSelector extends Component {
...
 
   render() {
+    const onSubmit = (e) => {
+      e.preventDefault();
+      this.bloodhound.search(this.typeahead.typeahead('val'), (suggestions) => {
+        if (suggestions.length) {
+          // Trigger the selection event on the first suggestion and close the
+          // typeahead
+          this.typeahead.trigger('typeahead:select', suggestions[0]);
+          this.typeahead.typeahead('close');
```
Comment (hbillings) [inline js/components/agency_component_selector.jsx]: yass, I was trying to figure out how to do this!

**Comment by hbillings** [comment]
@adborden That's awesome! I think we should merge so we can show the designers the interaction.

**Pull request merged**